src/Library/demos/Drop Down/main.js

```diff
@@ -0,0 +1,68 @@
+import Gtk from "gi://Gtk";
+import GObject from "gi://GObject";
+import Gio from "gi://Gio";
+
+const drop_down = workbench.builder.get_object("drop_down");
+const advanced_drop_down = workbench.builder.get_object("advanced_drop_down");
+
+drop_down.connect("notify::selected-item", () => {
+  const selected_item = drop_down.selected_item.get_string();
+  console.log(selected_item);
+});
+
+const key_value_pair = GObject.registerClass(
+  {
+    Properties: {
+      key: GObject.ParamSpec.string(
+        "key",
+        "Key",
+        "Key",
```

Comment:
```suggestion
        null,
        null,
```

I think that's less confusing for newcomers